src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesCount = Math.ceil(total / perPage);
+  const pages = getNumbers(1, pagesCount);
+  const firstPage = currentPage === 1;
+  const lastPage = currentPage === pagesCount;
```

Comment:
```suggestion
  const isFirstPage = currentPage === 1;
  const isLastPage = currentPage === pagesCount;
```

Reply:
please read this article about boolean variables names https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig